### PR TITLE
Internationalization Adaptation: Chinese China

### PR DESCRIPTION
**Pull request by conglinyizhi**
- [x]   1. I documented all behaviour within the code as far as I could.
- [x]   2. This PR targets the develop branch and *not* the master branch.
- [ ]   3. I have tested my changes extensively and paid extra attention to potential cross-platform issues (e./g. Cmd/Ctrl/Super key bindings)
- [ ]   4. I ran the `yarn lint` and `yarn test` commands successfully
- [x]   5. I have added an entry to the CHANGELOG.md
- [x]   6. I matched my code-style to the repository as far as possible
- [x]   7. I agree that my code will be published under the GNUP GPL v3 license
- [x]   8. In case I created new files, I added a header to them (see the existing files for examples)
- [x]   9. Before opening this PR, I ran `git pull` a last time to prevent any merge issues

I've only changed the internationalization file and a few changes to the code file, and I've only touched some of the code that isn't wrapped in the internationalization module, so I don't think it's necessary to run yarn lint and yarn test on my computer.
And I did try to see the results of my changes, but running the code directly from the dev branch gave me an error, and I've had similar problems with other projects I've contributed to, so I've ignored the problem for now in order to submit this code merge request properly.

## Description
Modify only the .po files required for internationalization, and those involved in code that does not wrap the text to be translated in `trans()`.

## Changes
It's just a change to the internationalization file and a small amount of .ts .vue code, and I don't think they're part of the API, and I don't think my changes are going to suddenly stop working on someone else's computer.

## Additional information
nodejs version is v20.9.0 (`node -v`)

### CODE REVIEW

**Comment by conglinyizhi** [comment]
: ( I saw your message, but I still don't think implementing yarn lint is necessary for internationalization contributions....

And I know you're a bot

File: source/app/service-providers/menu/menu.darwin.ts [hunk 1]
```diff
@@ -622,7 +622,7 @@ export default function getMenu (
         },
         {
           id: 'menu.new_window',
-          label: 'New window',
+          label: trans('New window'),
```
Comment (nathanlesage) [inline source/app/service-providers/menu/menu.darwin.ts:625]: Would you mind making this translatable also in the `menu.win32.ts`?.

**Review by nathanlesage — COMMENTED** [review]
Looks good, there is one additional place you could change as well, but other than that this looks fine and merge-able. Thanks!

**Comment by conglinyizhi** [comment]
Okay, now that I've made my changes, I'm waiting for your review and looking forward to applying my changes to the next release!

**Comment by nathanlesage** [comment]
Looks good! I'll just quickly wait for the checks to complete, and then I'll merge. Thank you for your contribution!

**Pull request merged**